RecoLocalTracker/SiPixelClusterizer/python/SiPixelClusterizerPreSplitting_cfi.py

```diff
@@ -10,9 +10,10 @@
     cpu = _siPixelClusters.clone()
 )
 
+from Configuration.Eras.Modifier_phase2_tracker_cff import phase2_tracker
 from Configuration.ProcessModifiers.gpu_cff import gpu
 # SwitchProducer wrapping the legacy pixel cluster producer or an alias for the pixel clusters information converted from SoA
-gpu.toModify(siPixelClustersPreSplitting,
+(gpu & ~phase2_tracker).toModify(siPixelClustersPreSplitting,
```

Comment:
here probably we don't want to remove the modifier for the cuda part... not sure how to do that though

Reply:
just split it into two `toModify` commands?

Reply:
done in the last push.